### PR TITLE
PP-13135: Cypress test for email notifications landing page

### PR DESCRIPTION
**Pull request by oswaldquek**

### CODE REVIEW

**Review by james-peacock-gds — CHANGES_REQUESTED** [review]
the code is great, just a tweak suggested.

File: test/cypress/integration/simplified-account/service-settings/email-notifications.cy.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    }
+  ]
+}
+
+const setupStubs = (role) => {
+  cy.task('setupStubs', [
+    userStubs.getUserSuccess({
+      userExternalId: USER_EXTERNAL_ID,
+      gatewayAccountId: GATEWAY_ACCOUNT_ID,
+      serviceName: { en: 'Trump based development' },
```
[the remark this answers is not in the document]
Comment (DomBelcher) [reply test/cypress/integration/simplified-account/service-settings/email-notifications.cy.js]: i've overwritten this in the git history

File: test/cypress/integration/simplified-account/service-settings/email-notifications.cy.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        cy.get('.govuk-summary-list').within(() => {
+          cy.get('.govuk-summary-list__key').eq(0).should('contain', 'Collect email addresses')
+          cy.get('.govuk-summary-list__value').eq(0).should('contain', 'On (mandatory)')
+          cy.get('.govuk-summary-list__actions a').eq(0).should('contain', 'Change')
+          cy.get('.govuk-summary-list__key').eq(1).should('contain', 'Payment confirmation emails')
+          cy.get('.govuk-summary-list__value').eq(1).should('contain', 'On')
+          cy.get('.govuk-summary-list__actions a').eq(1).should('contain', 'Change')
+          cy.get('.govuk-summary-list__key').eq(2).should('contain', 'Refund emails')
+          cy.get('.govuk-summary-list__value').eq(2).should('contain', 'On')
+          cy.get('.govuk-summary-list__actions a').eq(2).should('contain', 'Change')
```
Comment (DomBelcher) [inline test/cypress/integration/simplified-account/service-settings/email-notifications.cy.js:69]: I think we should test that these links go where they're expected to, for completeness

Comment (oswaldquek) [reply test/cypress/integration/simplified-account/service-settings/email-notifications.cy.js:69]: They don't go anywhere at the moment as the moment I've just done the [landing page](https://github.com/alphagov/pay-selfservice/pull/4297).

**Review by james-peacock-gds — APPROVED** [review]

**Pull request merged**